src/components/Navbar/Navbar.js

```diff
@@ -35,70 +36,14 @@ const Navbar = () => {
                 USD
```

Comment:
Hi Juancho!

I was reviewing the code and you made a great work saving the values in a constant. In the otherside, you have a hardcoded value here, I understand that this is the dropdown, but we have a problem here because if you select any other value, the dropdown will still showing the 'USD' value, so you can't use the value that we need. I think that you can save the selected value in a useState to save the value that we could need, also you could set a default value as 'USD' and them try if in that way you could change as many values that we want also you could try if the default value is showing at the start of the page.